lib/package-managers/npm.js

```diff
@@ -50,7 +67,26 @@ function view(packageName, field, currentVersion) {
     if (currentVersion && (!semver.validRange(currentVersion) || versionUtil.isWildCard(currentVersion))) {
         return Promise.resolve();
     }
-    const options = {agent, allVersions: true};
+    const options = {allVersions: true};
+
+    if (proxyAgent && noProxy !== true) {
+        options.agent = proxyAgent;
+
+        if (noProxy) {
+            const scope = packageName.includes('/') && packageName.split('/')[0];
+            const _registryUrl = registryUrl(scope);
+            const parsedRegistryUrl = URL ? new URL(_registryUrl) : url.parse(_registryUrl);
+            console.log(parsedRegistryUrl);
```

Comment:
Are these console.log's for debugging? If not, they should respond to `--loglevel`, like [`print`](https://github.com/tjunnone/npm-check-updates/blob/master/lib/npm-check-updates.js#L46-L52).

Reply:
Yep, you catched me in the middle of debugging the failing travis builds. Removed.